it/controllers/OtherReasonControllerISpec.scala

```diff
@@ -836,7 +838,9 @@ class OtherReasonControllerISpec extends IntegrationSpecCommonBase with FeatureS
       await(repository.updateStateOfFileUpload("1234", UploadJourney("file1", UploadStatusEnum.READY), isInitiateCall = true))
       val request = controller.removeFileUpload(NormalMode)(fakeRequest.withFormUrlEncodedBody("fileReference" -> "file1"))
       status(request) shouldBe SEE_OTHER
```

Comment:
It might be preferable to put this (`status(request) shouldBe SEE_OTHER`) in the eventually block, but up to you

Reply:
yup added